docs/reference/data-streams/data-streams.asciidoc

```diff
@@ -1,67 +1,59 @@
 [[data-streams]]
 = Data streams
+++++
+<titleabbrev>Data streams</titleabbrev>
+++++
 
-[partintro]
---
-You can use data streams to index time-based data that's continuously generated.
-A data stream groups indices from the same time-based data source.
-A data stream tracks its indices, known as _backing indices_, using an ordered
-list.
+A _data stream_ is a convenient, scalable way to ingest, search, and manage
+continuously generated time-series data.
 
-A data stream's backing indices are <<index-hidden,hidden>>.
-While all backing indices handle read requests, the most recently created
-backing index is the data stream's only write index.  A data stream only
-accepts <<docs-index_,index requests>> with `op_type` set to `create`. To update
-or delete specific documents in a data stream, submit a <<docs-delete,delete>>
-or <<docs-update,update>> API request to the backing index containing the
-document.
+Time-series data, such as logs, tends to grow over time, resulting in large
+volumes of data. While using a single {es} index for each data source is
+simpler, it is often more efficient and cost-effective to store a source's data
+across multiple, time-based indices. Multiple indices let you move indices
+containing older, less valuable data to less expensive hardware and delete
```

Comment:
Maybe replace "less valuable data" with "less frequently queried data"?

Reply:
Thanks. This wording is more explicit.